ros2doctor/ros2doctor/command/doctor.py

```diff
@@ -23,21 +24,31 @@ class DoctorCommand(CommandExtension):
     def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--report', '-r', action='store_true',
-            help='Print out doctor report.'
+            help='Print out all report info.'
+        )
+        parser.add_argument(
+            '--report_failed', '-rf', action='store_true',
+            help='Print out report info on failed checks.'
```

Comment:
It looks like `--report` and `--report_failed` are mutually exclusive options. Recommend using an [argparse mutually exclusive group](https://docs.python.org/3/library/argparse.html#mutual-exclusion) so the user gets an error when `--report` and `--report_failed` are both passed.